qemu/tests/live_backup_add_bitmap.py

```diff
@@ -53,7 +53,8 @@ def check_bitmap_existence_as_expected(bitmaps, existence_param):
     check_bitmap_existence_as_expected(bitmaps, "existence")
 
     error_context.context("system powerdown", test.log.info)
-    vm.monitor.system_powerdown()
+    #Fixme if the "guest refuse to go down" issue resolved
+    vm.monitor.quit()
```

Comment:
hi @aliang123 , it looks like a product bug here? 
If we work around the issue, we don't know if this issue has been fixed.
I mean it's expected to fail the case here

Reply:
@zhencliu, guest shutdown fail issue will be tracked by other cases, like shutdown.

Reply:
what about using vm.destroy() here, in which we do a graceful shutdown(cmd: shutdown -h now) first, then we call vm.monitor.quit() or kill the process if the graceful shutdown fails, it maybe a reasonable way to shutdown a vm for user

Reply:
@zhencliu, after considering the scenario, if we shutdown vm with kill, the presistent bitmap will become inconsistent, which is not the expected bitmap status. 
So if we can't shutdown or quit vm successfully, just failing them there is ok.

Reply:
I finally find a way to handle this:

if not vm.graceful_shutdown():
    vm.monitor.quit()

But the bad news is I found so many cases using vm.monitor.system_powerdown:

blockdev_inc_backup_disabled_persistent_bitmap.py:        self.main_vm.monitor.system_powerdown()
blockdev_inc_backup_inconsistent_bitmap.py:        self.main_vm.monitor.system_powerdown()
blockdev_inc_backup_mod_readonly_bitmap.py:        self.main_vm.monitor.system_powerdown()
blockdev_inc_backup_rm_persistent_bitmap.py:        self.main_vm.monitor.system_powerdown()
blockdev_mirror_ready_vm_down.py:        self.main_vm.monitor.system_powerdown()
.....